aspnetcore/security/anti-request-forgery.md

```diff
@@ -94,6 +94,7 @@ Antiforgery middleware is added to the [Dependency injection](xref:fundamentals/
 * <xref:Microsoft.AspNetCore.Builder.RazorPagesEndpointRouteBuilderExtensions.MapRazorPages%2A>
 * <xref:Microsoft.AspNetCore.Builder.ControllerEndpointRouteBuilderExtensions.MapControllerRoute%2A>
 * <xref:Microsoft.Extensions.DependencyInjection.RazorComponentsServiceCollectionExtensions.AddRazorComponents%2A>
+* [`AddAntiforgery`](/dotnet/api/microsoft.extensions.dependencyinjection.antiforgeryservicecollectionextensions.addantiforgery) <!-- <xref:Microsoft.Extensions.DependencyInjection.AntiforgeryServiceCollectionExtensions.AddAntiforgery(IServiceCollection)> zz -->
```

Comment:
See [Antiforgery middleware is added to the DI container when one of the following APIs is called](https://review.learn.microsoft.com/en-us/aspnet/core/security/anti-request-forgery?view=aspnetcore-8.0&branch=pr-en-us-30177#antiforgery-in-aspnet-core)